Assets/Scripts/Controllers/GameController.cs

```diff
@@ -46,70 +51,80 @@ public class GameController : MonoBehaviour {
         }
     }
 
-    void RunMenu()
+    private void RunMenu()
     {
-        if ( Input.GetKeyDown( KeyCode.Return ) ) {
+        if ( Input.GetKeyDown( KeyCode.Return ) )
+        {
             SceneController.LoadLevel( SceneController.Level.MAIN_LEVEL );
+            UnPauseGame();
         }
-
-        Time.timeScale = 0.0f;
     }
 
-    void RunGame()
-    {
-        Time.timeScale = 1.0f;
-    }
+    private void RunGame() {}
 
-    void RunGameEnd()
+    private void RunGameEnd()
     {
-        if ( Input.GetKeyDown( KeyCode.Return ) ) {
+        if ( Input.GetKeyDown( KeyCode.Return ) )
+        {
             RestartGame();
         }
     }
 
     private void RestartGame()
     {
         // Reset game
-        PlayController.instance.Restart();
+        PlayController.pcInstance.Restart();
         SceneController.LoadLevel( SceneController.Level.MAIN_LEVEL );
-
-        // Unfreeze time
-        Time.timeScale = 1.0f;
+        UnPauseGame();
     }
 
-    public void GameOver(int score)
+    public void EndGame(bool wonGame, int score=0)
     {
-        SceneController.LoadLevel(SceneController.Level.GAME_OVER_MENU);
-        UIController.uicInstance.SetFinalScore(score);
+        isGameOver = true;
 
-        Time.timeScale = 0.0f;
+        if (wonGame)
+        {
+            SceneController.LoadLevel(SceneController.Level.WIN_MENU);
+        }
+        else
+        {
+            SceneController.LoadLevel(SceneController.Level.GAME_OVER_MENU);
+            UIController.uicInstance.SetFinalScore(score);
+        }
+
+        PauseGame();
     }
 
-    public void GameWon()
+    private void PauseGame()
     {
-        SceneController.LoadLevel(SceneController.Level.WIN_MENU);
+        if ( !isPaused )
+        {
+            isPaused = true;
+            Time.timeScale = 0.0f;
 
-        Time.timeScale = 0.0f;
+            // Set default cursor
+            InputController.Instance.SetDefaultCursor();
```

Comment:
This was me making it so when we pause we see default cursor.

Eventually we will have to address some of these cursor based changes if we have menus with buttons for example.
Or if we end up not having a cursor at all for mobile